### ISSUE TITLE
Way too many non-core dependencies [rt.cpan.org #71661]

### ISSUE THREAD

**Issue by xdg** [cpan-testers/CPAN-Reporter#65]
https://rt.cpan.org/Ticket/Display.html?id=71661

    Hi, 
    
    CPAN::Reporter has way too many dependencies. Also, before any reports 
    can be sent, Test::Reporter::Transport::Metabase is required and that 
    has even more dependencies, by an order of magnitude. I'm talking about 
    setting up a fresh install of the perl interpreter and a clean, new 
    lib/.
    
    All the time that these dependencies are installing, warnings are 
    emitted from CPAN::Reporter. At the least, these should be suppressed. I 
    suppose I could 'o conf' cpan and disable; but why? 
    
    Anyway, no reports are going to be sent during installation of this 
    module and its dependencies. Suggest cutting back on the deps. 
    
    XIONG@cpan.org

**Comment by eserte** [comment]
In the original RT ticket I offered a script sending filed reports, created on a system without a Metabase client. Here it is:
https://github.com/eserte/srezic-misc/blob/master/scripts/send_tr_reports.pl

**Comment by jkeenan** [comment]
> https://rt.cpan.org/Ticket/Display.html?id=71661
> 
> ```
> Hi, 
> 
> CPAN::Reporter has way too many dependencies. Also, before any reports 
> can be sent, Test::Reporter::Transport::Metabase is required and that 
> has even more dependencies, by an order of magnitude. I'm talking about 
> setting up a fresh install of the perl interpreter and a clean, new 
> lib/.
> 
> All the time that these dependencies are installing, warnings are 
> emitted from CPAN::Reporter. At the least, these should be suppressed. I 
> suppose I could 'o conf' cpan and disable; but why? 
> 
> Anyway, no reports are going to be sent during installation of this 
> module and its dependencies. Suggest cutting back on the deps. 
> 
> XIONG@cpan.org
> ```

Yes, I wish that the prerequisites were fewer in number and loaded more quickly.  But I suspect that most of the major time-savers were explored at Perl QA hackathons over the years.

@eserte, in a reply in the RT ticket from which this GH issue was transferred, supplied a reasonable work-around for the original poster's problem.  Since we haven't heard back from the OP at all, I proposed that we close this ticket.
